hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataNode.java

```diff
@@ -865,7 +942,7 @@ private void refreshVolumes(String newVolumes) throws IOException {
               .newFixedThreadPool(changedVolumes.newLocations.size());
           List<Future<IOException>> exceptions = Lists.newArrayList();
 
-          Preconditions.checkNotNull(data, "Storage not yet initialized");
+          checkNotNull(data, "Storage not yet initialized");
```

Comment:
let the imports stay as is. no need to bother them

Reply:
Thanks. I will fix this.